svgdigitizer/__main__.py

```diff
@@ -278,6 +275,31 @@ def digitize_cv(svg, sampling_interval, metadata, package, outdir, skewed):
         _write_metadata(json, package.descriptor if package else cv.metadata)
 
 
+def _create_package(metadata, csvname, outdir):
+    r"""
+    Return a data package built from a :param:`metadata` dict and tabular data
+    in :param:`csvname`.
+
+    This is a helper method for :meth:`digitize_cv`.
+    """
+    from datapackage import Package
+
+    package = Package(cv.metadata, base_path=outdir or os.path.dirname(csvname))
+    package.infer(os.path.basename(csvname))
+    # Update fields describing the data in the CSV
+    package_fields = package.descriptor["resources"][0]["schema"]["fields"]
+    data_description_fields = package.descriptor["data description"]["fields"]
+
+    new_fields = []
+    for idx, field in enumerate(package_fields):
+        if field["name"] == data_description_fields[idx]["name"]:
```

Comment:
This is correct. As far as I understand your approach (and mine), the field items have to be in the same order. I will address this in a separate PR. It might be convenient to use frictionless and its field class.

Reply:
Ok. So I think what you are doing here is wrong ;)

Reply:
This will be addressed in #142